From: Luca Beltrame <lbeltrame@kde.org>
Date: Fri, 04 Jan 2013 19:31:12 +0000
Subject: Use a slightly better strategy for symlinking: although the previous
X-Git-Url: http://quickgit.kde.org/?p=pykde4.git&a=commitdiff&h=4e8ee28f3a314edef2a8489854c0ed58ac668070
---
Use a slightly better strategy for symlinking: although the previous
code working locally, distribution users had issues in it not properly
working.

Patch by Raymond Wooninck.

CCMAIL: kde-packager@kde.org
---


--- a/tools/pykdeuic4/CMakeLists.txt
+++ b/tools/pykdeuic4/CMakeLists.txt
@@ -10,7 +10,7 @@
 
 if (NOT PYKDEUIC4_ALTINSTALL)
     # Step 2: Make a the generated symlink a link to "pykdeuic4"
-    install(CODE "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink ${BIN_INSTALL_DIR}/${_uic_name} ${BIN_INSTALL_DIR}/pykdeuic4)" )
+    INSTALL(CODE "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -DTARGET=${BIN_INSTALL_DIR}/${_uic_name} -DLINK_NAME=${BIN_INSTALL_DIR}/pykdeuic4 -P ${cmake_module_dir}/create_exe_symlink.cmake)" )
 endif()
 
 unset(_uic_name)

